Engine/PyParameter.h

```diff
@@ -428,6 +428,7 @@ class Int2DParam
 
     Int2DTuple get() const;
     Int2DTuple get(double frame) const;
+    using IntParam::set;
```

Comment:
why?

Reply:
Both `Int2DParam` and `Int3DParam` require a function definition of `get(int)` (`get(int, int)` too for 3D) if not it will be shadowed, another way to solve this is by qualifying the method calls but is way harder to do so. See https://stackoverflow.com/questions/24342270/base-class-template-member-function-shadowed-in-derived-class-albeit-different

Reply:
Why do we need these functions? I think it's expected to have them shadowed, and I don't see what this has to do with this PR

Reply:
If hiding is expected then Shiboken2 was doing the very nasty thing of ignoring the shadowing and generating the `get(int)` where it wasn't supposed to be. IIRC these kind of exclusions can be done in the typesystem.

Reply:
Then I would say a proper solution would be to provide explicit implementations that throw an exception. Don't you think so?

Reply:
Sounds like it's the most sanish way to do so. One way could be immediately throwing a mismatched dimension error. Another putting those methods under a `private` field although at the cost of reporting a wrong error.